statistics/handle/cache/internal/lfu/key_set.go

```diff
@@ -0,0 +1,59 @@
+// Copyright 2023 PingCAP, Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package lfu
+
+import (
+	"sync"
+
+	"golang.org/x/exp/maps"
+)
+
+type keySet struct {
+	set map[int64]struct{}
+	mu  sync.RWMutex
+}
+
+func (ks *keySet) Add(key int64) {
+	ks.mu.Lock()
+	ks.set[key] = struct{}{}
+	ks.mu.Unlock()
+}
+
+func (ks *keySet) Remove(key int64) {
+	ks.mu.Lock()
+	delete(ks.set, key)
+	ks.mu.Unlock()
+}
+
+func (ks *keySet) Contains(key int64) bool {
+	ks.mu.RLock()
+	_, ok := ks.set[key]
+	ks.mu.RUnlock()
+	return ok
+}
```

Comment:
No place uses this method, can we remove it? Better to keep our implementation as simple as possible.

Reply:
Removed.